src/mudlet.h

```diff
@@ -326,6 +326,8 @@ class mudlet : public QMainWindow, public Ui::main_window
     void refreshTabBar();
 
     bool firstLaunch = false;
+    // mirror everything shown in any console to stdout. Helpful for CI environments
+    bool mirrorToStdout = false;
```

Comment:
This is currently a `pubic` member of a class that is used as a singleton and is only set at the start when the command line is parsed - so it is *almost* a run-time constant. Would it not be better to make it a, e.g. `static private smMirrorToStdOut` and then provide access via a (`static`) getter, e.g.:
```cpp
    static bool getMirrorState() {return smMirrorToStdOut;}
```
then you do not need to create and initialise a `mirrorToStdout` in ***every*** `TConsole` instance (including *buffers* BTW) and replace the current places where it is being tested with a `mudlet::getMirrorState()`?

Reply:
I thought about the latency that would have, but I suppose that won't be an issue right?

Reply:
I don't *think* so - but then I am not an expert...

(I guess Ahmed Charles might be!)